src/core/js/models/adaptModel.js

```diff
@@ -236,9 +245,9 @@ define([
         },
 
         getAvailableChildren: function() {
-            return new Backbone.Collection(this.getChildren().where({
+            return this.getChildren().where({
```

Comment:
I can go either way. I see the advantage of tandem functions as it will require less work for plugin authors for them to pull up to v3.